backend/src/components/application.js

```diff
@@ -447,7 +447,8 @@ async function printPdf(req, numOfRetries = 0) {
       '--disable-software-rasterizer',
       '--disable-dev-shm-usage',
       '--disable-gpu',
-    ]
+    ],
+    userDataDir: '/tmp/chromium'
```

Comment:
Does this directory get created as part of the chromium install?

Reply:
@weskubo-cgi The runtime creates it in the /tmp directory, where any user may have permission to create folders for disposable data.  If the directory were created in something like the Dockerfile, it will have root ownership and break the runtime.